packages/web/src/components/status-message/StatusMessage.tsx

```diff
@@ -0,0 +1,110 @@
+import { CSSProperties } from 'react'
```

Comment:
This is an old component. In the process of creating the Harmony `CompletionCheck` component, I converted it to Typescript.